tensorflow/python/kernel_tests/signal/fft_ops_test.py

```diff
@@ -554,6 +554,47 @@ def testGrad_Random(self):
             self._checkGradComplex(
                 self._tfIFFTForRank(rank), re, im, result_is_complex=False)
 
+class FFTShiftTest(test.TestCase):
```

Comment:
please use self.assertAllEqual instead of self.assertTrue((a == b).all()) since it'll give better error messages.